frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const serverUrl = 'http://localhost:3006';
+
+export async function addPoint(teamId) {
+  try {
+    await axios.post(`${serverUrl}/addPoint/${teamId}`);
+  } catch (error) {}
+}
```

Comment:
I can see both #65 and #67 just swallow their exceptions. Wouldn't it be nice to at least show an `alert`? Not most civilized, but at least it doesn't deceive users into thinking that no error has happened and it would do for now.